yarn.lock

```diff
@@ -24506,20 +24506,6 @@ react-markdown@^4.0.6:
     unist-util-visit "^1.3.0"
     xtend "^4.0.1"
 
-react-markdown@^4.2.2:
```

Comment:
Description says "Moves over / creates package.json file for ingest_manager" but I don't see a new package.json and this appears to be deleting the EPM-specific version

Reply:
oops, thanks!